[{"commit":{"message":"Adding CancellationException detail message to CompletableFuture"},"files":[{"filename":"src\/java.base\/share\/classes\/java\/util\/concurrent\/CompletableFuture.java"},{"filename":"test\/jdk\/java\/util\/concurrent\/tck\/CompletableFutureTest.java"}],"sha":"078930abee311cfdda2cf84a2cb91ea5d82ba172"},{"commit":{"message":"Make it possible to customize the message when wrapping a CancellationException"},"files":[{"filename":"src\/java.base\/share\/classes\/java\/util\/concurrent\/CancellationException.java"},{"filename":"src\/java.base\/share\/classes\/java\/util\/concurrent\/CompletableFuture.java"}],"sha":"641bbfe36982337d7fa8ac29fdc09fbdd332ed11"},{"commit":{"message":"Adding wrapping of CompletableFuture CancellationExceptions from get() and join() to add diagnostic information"},"files":[{"filename":"src\/java.base\/share\/classes\/java\/util\/concurrent\/CancellationException.java"},{"filename":"src\/java.base\/share\/classes\/java\/util\/concurrent\/CompletableFuture.java"},{"filename":"test\/jdk\/java\/util\/concurrent\/tck\/CompletableFutureTest.java"}],"sha":"2ef53fd264571fdf5911bc4ae6a868a794777a4d"}]